Updates to the Single DMP notebook

briri opened the pull request:
- Display award URI in the tooltip
- Update query to extract ProjectLeaders and DataCurators instead of Contributor with `nameType="Personal"` (EZID does not seem to be passing this through)
- remove HostingInstitution (won't be relevant for this project)
- Small typo fix

kjgarza commented on dmp/user-story-single-dmp-connections.ipynb:
```diff
@@ -166,11 +166,6 @@
     "          }\n",
```
<p>I think you would still need to remove filter_people from inside the create_node execution. Here it will filter by name type for three arrays of results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

briri replied:
Yes, forgot to apply that to the changes here. I have removed that function

kjgarza commented on dmp/user-story-single-dmp-connections.ipynb:
```diff
@@ -166,11 +166,6 @@
     "          }\n",
```
<p>in line 150, Datum.award, i don’t think it exists in the query, so it will always return null. </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

briri replied:
Ah yes, I missed that when moving my changes from within the binder site to the git repo. I added 'award: awardUri' to the query.

briri commented on dmp/user-story-all-dmps.ipynb:
```diff
@@ -134,10 +134,12 @@
    "source": [
     " # Generate the GraphQL query to retrieve up to 100 outputs of University of Oxford, with at least 100 views each.\n",
+    "repo_id = \"cdl.cdl\" if f.value == \"https://ror.org/03yrm5c26\" else \"cern.zenodo\"\n",
+    "funder_id = \"https://doi.org/10.13039/100000141\" if f.value == \"https://ror.org/03yrm5c26\" else \"https://doi.org/10.13039/501100000780\"\n",
```
would be nice to allow for a collection of funders here. In particular we have 'NSF' (10.13039/100000001) and a sub-group within NSF, the Division of Ocean Sciences (10.13039/100000141). I didn't see a way to do that below. I tried `funders (containsId: funder_id)` but received errors.

briri commented:
@kjgarza I've addressed the issues you noted with the single-dmp notebook and also made some changes to the all-dmps notebook to update the funder and repo ids used based on the user's selection for Org. 

The all-dmps doesn't provide us with any publication or dataset info. I'm guessing this is because its looking for the DMP DOI to be a relatedIdentifier on their records. Is that an accurate assumption? 

I'm not sure why the funder query never returns anything for either the 'NSF' or 'Divison of Ocean Sciences' funder ids. If you select the CDL ror and run the notebook, you can see those funders listed in the org and repo tables but I cannot get the fuunder one to return anything.

kjgarza commented on dmp/user-story-all-dmps.ipynb:
```diff
@@ -134,10 +134,12 @@
    "outputs": [],
```
<p>https://doi.org/10.13039/100000141 this funder_id is from "Division of Ocean Sciences" and has zero management plans, is that the correct DOI ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

briri replied:
Yes, for example 'doi:10.48321/D14S38' has:
`<fundingReference><funderName>Division of Ocean Sciences (nsf.gov)</funderName><funderIdentifier funderIdentifierType="Crossref Funder ID">https://api.crossref.org/funders/100000141</funderIdentifier><awardNumber awardURI="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1356966">1356966</awardNumber><awardTitle>Turbulence-spurred settlement: Deciphering a newly recognized class of larval response</awardTitle></fundingReference>`

The funder is listed in the Organization and repository tables.
